### PR TITLE
RxJS/Angular/Web Speech post - part 1 of 2

### PR DESCRIPTION
**Pull request by kmaida**
Publication: scheduled for 9/20/2017

### CODE REVIEW

File: _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown [hunk 1]
```diff
@@ -0,0 +1,1294 @@
...
+This is **Part 1** of our 2-part tutorial. Part 1 will cover the following:
+
+* Introduction to reactive programming with Rx* and observables
+* Setting up our Angular app
+* Integrating the Annyang library for speech recognition support
+* Implementing a form component allowing users to speak or type words to use in the madlib story
+
+In Part 2, we will cover:
+
+* Keyboard component fallback if user's browser does not support speech recognition
```
Comment (unicodeveloper) [inline _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown]: Please add a full stop at the end of each bullet points.

File: _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown [hunk 2]
```diff
@@ -0,0 +1,1294 @@
...
+---
+
+**TL;DR:** In this 2-part tutorial series, we'll learn how to build a small app with some big concepts. We'll cover reactive programming with [Reactive Extensions (Rx*)](http://reactivex.io/), JS framework component interaction in [Angular](https://angular.io), and speech recognition with the [Web Speech API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API). The completed Madlibs app code can be found at [this GitHub repo](https://github.com/auth0-blog/madlibs).
+
+---
+
+This is **Part 1** of our 2-part tutorial. Part 1 will cover the following:
+
+* Introduction to reactive programming with Rx* and observables
+* Setting up our Angular app
```
Comment (unicodeveloper) [inline _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown]: Please add a full stop at the end of each bullet points.

File: _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown [hunk 3]
```diff
@@ -0,0 +1,1290 @@
...
+Let's get started!
+
+## What We'll Learn About
+
+The concepts and tools we'll use and learn about include:
+
+* Component communication in [Angular](https://angular.io) JavaScript framework
+* Managing streams with [RxJS](http://reactivex.io/rxjs/) ReactiveX for JavaScript (packaged with Angular)
+* [Annyang](https://github.com/TalAter/annyang) library for [Web API Speech Recognition](https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition)
+* Accessing a prebuilt [Node API](https://github.com/kmaida/madlibs-api) to generate random words
```
[the remark this answers is not in the document]
Comment (kmaida) [reply _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown]: These aren't actually sentences, they're just list items, so it isn't semantically correct to add a full stop unless the copy is changed. Do you recommend that the content be edited to make these full sentences?

File: _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown (hunk 2 above)
Comment (kmaida) [reply _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown]: (same comment as above re: list items, not sentences)

File: _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown (hunk 1 above)
Comment (kmaida) [reply _posts/2017-09-20-going-in-depth-with-rxjs-angular-web-speech-part-1.markdown]: (same comment as above re: list items, not sentences)

**Pull request merged**